frontend/src/components/object/ObjectTable.vue

```diff
@@ -9,7 +9,8 @@ import {
   ObjectFilters,
   ObjectPermission,
 } from '@/components/object';
-import { ShareObjectButton, SyncObjectButton } from '@/components/object/share';
+import { ShareObjectButton } from '@/components/object/share';
+import { SyncButton } from '@/components/common';
```

Comment:
Switch the import ordering - we sort based on the from string, not the component name.